Northstar.Client/mod/scripts/vscripts/ui/menu_ns_modmenu.nut

```diff
@@ -711,4 +711,5 @@ void function ReloadMods()
 
 	// note: the logic for this seems really odd, unsure why it doesn't seem to update, since the same code seems to get run irregardless of whether we've read weapon data before
 	ClientCommand( "uiscript_reset" )
-}
```

Comment:
what

Reply:
IDE automatically added trailing newline, i.e. `}` vs `}\n`.

Not a deal breaker.